static/js/lightbox.js

```diff
@@ -8,6 +8,24 @@ var asset_map = {
 
 };
 
+// save the canvas preferences for the lightbox in the "lightbox-canvas-settings"
+// key.
+var ls = localstorage();
+ls.version = 1;
+
+var KEY = "lightbox-canvas-settings";
+
+var lightbox_settings = ls.get(KEY);
+
+if (!lightbox_settings) {
+    lightbox_settings = {
+        canvas_enabled: false,
+        zoom_speed: 1,
+    };
+
+    ls.set(KEY, lightbox_settings);
+}
```

Comment:
Does this correctly handle `localStorage` not being available in the browser?  We've had a bunch of bugs elsewhere with not checking `localStorage.is_enabled()`